hedera-node/docs/design/services/token-service/airdrops/airdrop-transaction.md

```diff
@@ -211,4 +212,5 @@ If the airdrop transaction fees from the list above are covered by a separate pa
 * Verify that an airdrop, which fee expenses are covered by a separate payer account, but has missing payer signature fails
 * Verify that an airdrop with more than 10 entries in the `token_transfers` list fails
 * Verify that an airdrop with a duplicate entry for the same NFT in the `token_transfers` list fails with `PENDING_NFT_AIRDROP_ALREADY_EXISTS`
+* Verify that deletion of an account who has sent an airdrop that is still in pending state fails
```

Comment:
This probably should not fail (the account owner must be able to delete the account without resorting to a third-party action).  I believe it should delete the pending airdrop instead, so that a pending airdrop cannot "pin" an account in place.

Reply:
We had some discussions that in order to delete an account that has sent airdrops, you should first delete all related airdrops to this account. So, I believe this should happen with explicit TokenAirdropCancel transactions beforehand.

@Nana-EC, @netopyr  what do you think?

Reply:
The recipient account cannot cancel the airdrops, so a token holder could, in effect, pin someone else's account by sending airdrops that would have to be claimed (and using a structure that makes claim unpalatable).

Reply:
I misread the line as a recipient rather than a sender.

After re-reading, I agree with the existing wording.